server/src/main/java/org/apache/uniffle/server/buffer/ShuffleBufferManager.java

```diff
@@ -125,6 +126,8 @@ public ShuffleBufferManager(
     this.bufferFlushEnabled = conf.getBoolean(ShuffleServerConf.SINGLE_BUFFER_FLUSH_ENABLED);
     this.bufferFlushThreshold =
         conf.getSizeAsBytes(ShuffleServerConf.SINGLE_BUFFER_FLUSH_THRESHOLD);
```

Comment:
We should rename this to
```suggestion
        conf.getSizeAsBytes(ShuffleServerConf.SINGLE_BUFFER_FLUSH_SIZE_THRESHOLD);
```
to better reflect the difference of this threshold to `SINGLE_BUFFER_FLUSH_BLOCKS_NUM_THRESHOLD`.

Reply:
OK. I'll change this. At first, I thought about not affecting other code, so I didn't make any changes. Indeed, the naming here is not very reasonable after this PR.